cluster/aws/templates/common.sh

```diff
@@ -15,11 +15,15 @@
 # limitations under the License.
 
 
-apt-get update
-apt-get install --yes curl
+until apt-get update; do
+  echo "== apt-get update failed, retrying =="
+  echo sleep 5
+done
+
+apt-get install -y curl
+apt-get dist-upgrade -y
 
 # Retry a download until we get it.
-#
```

Comment:
I honestly at the time thought it was ridiculous to put one word on the next line when the comments are stripped anyway